engine/src/main/java/org/hibernate/search/bridge/spi/FieldMetadataCreationContext.java

```diff
@@ -33,4 +33,12 @@
 	 * @return this context for fluent API invocations
 	 */
 	FieldMetadataCreationContext sortable(boolean sortable);
+
+	/**
+	 * Give access to properties that are backend specific
```

Comment:
"Give access to properties that are specific to an indexing technology" ? Not particularly fond of that, I'm open to better ideas.